src/main/java/kroppeb/stareval/parser/Op.java

```diff
@@ -0,0 +1,8 @@
+package kroppeb.stareval.parser;
+
+
+abstract class Op {
+	
+	@Override
+	abstract public String toString();
```

Comment:
This might be incorrect codestyle, as many classes (Such as GlResource) use `public abstract` rather than `abstract public`, will leave comments on all occurrences